### PR TITLE
Re-enable extensions where fixes have been applied

### PR DESCRIPTION
**Pull request by microbit-mark**
Re-enable extensions where fixes have been applied.

We can add these per extension/vendor in future, just getting through the first few.

- https://github.com/microsoft/pxt-microbit/issues/3736
- https://github.com/microsoft/pxt-microbit/issues/3751
- https://github.com/microsoft/pxt-microbit/issues/3750
- https://github.com/elecfreaks/pxt-PlanetX/issues/5
- https://github.com/bsiever/microbit-pxt-timeanddate/issues/11

### CODE REVIEW

File: targetconfig.json [hunk 1]
```diff
@@ -221,11 +221,11 @@
             "alsrobot-microbit-makecode-packages/cruisebit": "dv:mbcodal",
             "tinkertanker/udriver_pca9585": "dv:mbcodal",
             "dfrobot/pxt-dfrobot-naturalscience": "dv:mbcodal",
-            "kitronikltd/pxt-kitronik-zip-tile": "min:1.0.2",
-            "4tronix/bitcommander": "min:1.0.2",
+            "kitronikltd/pxt-kitronik-zip-tile": "min:0.1.0",
```
Comment (abchatra) [inline targetconfig.json:224]: This should be min:v0.1.0 right? Tag has _v_ in front of it. Please check the others as well.

**Review by abchatra — CHANGES_REQUESTED** [review]

File: targetconfig.json (hunk 1 above)
Comment (microbit-mark) [reply targetconfig.json:224]: I was taking the version from https://github.com/KitronikLtd/pxt-kitronik-zip-tile/blob/85d5c24d15648458770ea47b0b61949cd65a3581/pxt.json#L3 should it be the tagged release as in https://github.com/KitronikLtd/pxt-kitronik-zip-tile/releases/tag/v0.1.0 ?

Comment (microbit-mark) [reply targetconfig.json]: Updated in https://github.com/microsoft/pxt-microbit/pull/3769/commits/5af55f737e976437578367c14f432d6751bb7ce2

Comment (microbit-mark) [reply targetconfig.json]: @abchatra is this now ok?

**Review by abchatra — APPROVED** [review]

**Review by abchatra — APPROVED** [review]

**Review by abchatra — APPROVED** [review]

**Pull request merged**